content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -188,6 +188,27 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
   const router = useRouter();
   const [stopNumber, setStopNumber] = useState(stopNumberServerSide);
   const [currentStop, setCurrentStop] = useState(currentStopServerSide);
+  const [headerEl, setHeaderEl] = useState<HTMLElement>();
+
+  const headerRef = useCallback((node: HTMLElement) => {
+    if (node) setHeaderEl(node);
+  }, []);
+
+  useEffect(() => {
+    if (!headerEl) return;
+    // We measure the height of the Header element with a ResizeObserver and
+    // update the sticky top position of the StickyPlayer element any time it
+    // changes
+    const resizeObserver = new ResizeObserver(([entry]) => {
+      document.documentElement.style.setProperty(
+        '--stop-header-height',
+        `${entry.contentRect.height}px`
+      );
+    });
+    resizeObserver.observe(headerEl);
+
+    return () => resizeObserver.disconnect();
+  }, [headerEl]);
 
   useEffect(() => {
     setStopNumber(Number(router.query.stop));
```

Comment:
You might not need it here (I can't run this locally right now as I'm deep in uncommited code), but wondering if passing this through [isMaybeNumber](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/utils/routes.ts#L95) would be good in order to avoid `string[]` formats? (Might be overkill)

Reply:
I meant `isMaybeString`, amended the comment.

Just because I've been doing that a lot in the redirections bc of TS so it's at the front of my brain. 

I do 

```
  const validStopNumber = Number(toMaybeString(stopNumber))
      ? toMaybeString(stopNumber)
      : undefined;
```

which I currently find maybe intense but would like your thoughts on?

Reply:
I feel like this would be overkill here, because if we can't find a `currentStopServerSide` then we send a 404? I might be missing something though

Reply:
OK cool! Like I said I've just been way too knee deep in this 😄 makes sense 👍